plugins.js

```diff
@@ -427,13 +427,15 @@ plugins.run_next_hook = function (hook, object, params) {
         object.logdebug('aborting ' + hook + ' hook');
         return;
     }
-    var called_once = false;
-    var timeout_id;
-    var timed_out = false;
-    var cancelled = false;
-    var cancel = function () { cancelled = true; };
-    var item;
-    var callback = function (retval, msg) {
+    let called_once = false;
+    let timeout_id;
+    let timed_out = false;
+    let cancelled = false;
+    /* eslint-disable prefer-const */
```

Comment:
If we're going to disable eslint stuff, wouldn't it be better if we at least put this at the top of the file?